lib/framework/elastic_connector_actions.rb

```diff
@@ -12,88 +12,137 @@
 module Framework
   class ElasticConnectorActions
     CONNECTORS_INDEX = '.elastic-connectors'
+    JOB_INDEX = '.elastic-connectors-sync-jobs'
 
-    def self.force_sync(connector_package_id)
-      body = {
-        :doc => {
-          :scheduling => { :enabled => true },
-          :sync_now => true
+    class << self
+
+      def force_sync(connector_package_id)
```

Comment:
While I agree with refactoring (`class << self`), I think we should avoid having multiple various changes in a single PR - it should be split to a different PR and reviewed separately.

I see it as a recurring pattern - when PR is small and starts to expand, adding multiple various changes which slows down the review/merge procedure + increase the changes/size of merge conflicts.

Reply:
Agree. Will avoid doing so.